app/scripts/app-init.js

```diff
@@ -12,10 +12,8 @@ function tryImport(...fileNames) {
 
 function importAllScripts() {
   const startImportScriptsTime = Date.now();
-  // applyLavaMoat has been hard coded to "true" as
-  // tryImport('./runtime-cjs.js') is giving issue with XMLHttpRequest object which is not avaialble to service worker.
-  // we need to dynamically inject values of applyLavaMoat once this is fixed.
-  const applyLavaMoat = true;
+  // value of applyLavaMoat below is dynamically replaced at build time with actual value
+  const applyLavaMoat = false;
```

Comment:
default to true (always default to safe)

Reply:
updated now